retrofit/src/main/java/retrofit/OkHttpCall.java

```diff
@@ -157,4 +158,24 @@ public void cancel() {
       rawCall.cancel();
     }
   }
+
+  @Override
+  public String toString() {
+    return "OkHttpCall{"
+        + "retrofit="
+        + retrofit
+        + ", requestFactory="
+        + requestFactory
+        + ", responseConverter="
+        + responseConverter
+        + ", args="
+        + Arrays.toString(args)
```

Comment:
This is troublesome as a password might be an argument to a service method. By moving its string value into an exception it could be inadvertently logged or passed to untrusted code.